algorithms/Java/recursion/fibonacci.java

```diff
@@ -0,0 +1,18 @@
+package com.dsa;
```

Comment:
Please add this file reference on this readme [file](https://github.com/Satyap1766/DSA/blob/main/algorithms/Java/README.md) using relative link